LMS/User/Http/Controllers/MeController.php

```diff
@@ -16,9 +17,9 @@ public function __construct(MeRepository $repository)
         $this->repository = $repository;
     }
 
-    public function putMe(UpdateMeRequest $request): Response
+    public function putMe(UpdateMeRequest $request): JsonResponse
     {
         $this->repository->updateBaseInformation($request->validated());
-        return response()->noContent();
+        return response()->json(['message' => 'Informações atualizadas com sucesso!'],Response::HTTP_OK);
```

Comment:
Deixe um espaço entre os parâmetros sempre que possível! É bem comum alguém que estiver usando identador automático acabar tendo que commitar essas linhas pq esqueceu alguma convenção de codiguin bonito.

```suggestion
        return response()->json(['message' => 'Informações atualizadas com sucesso!'], Response::HTTP_OK);
```